R/pkg/R/DataFrame.R

```diff
@@ -1869,6 +1869,7 @@ setMethod("where",
 #' path <- "path/to/file.json"
 #' df <- read.json(path)
 #' dropDuplicates(df)
+#' dropDuplicates(df, "col1", "col2")
 #' dropDuplicates(df, c("col1", "col2"))
 #' }
 setMethod("dropDuplicates",
```

Comment:
Since we have unified the generics, can we also fold this method into the other one ? i.e. move the code here to the `length(col) > 1` case in the other function